autorest/client.go

```diff
@@ -72,6 +74,52 @@ const (
 `
 )
 
+type logLevelType int
+
+const (
+	logLevelNone logLevelType = iota
+	logLevelBasic
+	logLevelFull
+)
+
+const (
+	logTimestamp = "2006-01-02T15:04:05.0000000Z07:00"
```

Comment:
I'd add a comment here just to remind any new gophers spinning up on the team that this date/time is semantically important.